docs/src/pages/components/lists/InsetList.tsx

```diff
@@ -13,15 +13,15 @@ export default function InsetList() {
       component="nav"
```

Comment:
Would it be better to render ul > li in this context?

Reply:
the children are `ListItemButton` which render `div` by default. What is the different here?

`nav > div` or `ul > li`

In this case, it sounds like a `nav` to me but either works I think.

Reply:
IMHO nav > ul > li > button is a superior DOM structure for navigation.

Reply:
updated the demo.